ste/xfer-remoteToLocal.go

```diff
@@ -53,14 +53,19 @@ func remoteToLocal(jptm IJobPartTransferMgr, p pipeline.Pipeline, pacer pacer, d
 	// then check the file exists at the remote location
 	// if it does, react accordingly
 	if jptm.GetOverwriteOption() != common.EOverwriteOption.True() {
-		_, err := os.Stat(info.Destination)
+		dstProps, err := os.Stat(info.Destination)
 		if err == nil {
 			// if the error is nil, then file exists locally
 			shouldOverwrite := false
 
 			// if necessary, prompt to confirm user's intent
 			if jptm.GetOverwriteOption() == common.EOverwriteOption.Prompt() {
 				shouldOverwrite = jptm.GetOverwritePrompter().shouldOverwrite(info.Destination)
+			} else if jptm.GetOverwriteOption() == common.EOverwriteOption.IfSourceNewer() {
+				// only overwrite if source lmt is newer (after) the destination
+				if jptm.LastModifiedTime().After(dstProps.ModTime()) {
+					shouldOverwrite = true
```

Comment:
I suspect the same issue applies here as I mentioned for the anyToRemote case, and again the safest short-term solution appears to be to retrive the latest LMT here, using sourceInfoProvider (check that they all requery and don't rely on the scan-time result!)

Reply:
Resolving. My logic was incorrrect (or at least, unnecessary).